React/Views/RCTViewManager.m

```diff
@@ -275,4 +275,14 @@ - (RCTViewEventHandler)eventHandlerWithName:(NSString *)eventName json:(id)json
 
 RCT_REMAP_SHADOW_PROPERTY(onLayout, hasOnLayout, BOOL)
 
+RCT_CUSTOM_SHADOW_PROPERTY(aspectRatio, CGSize, RCTShadowView) {
```

Comment:
I believe you should handle the case where `json` is nil with `defaultView.aspectRatio`.